java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -56,6 +58,91 @@ public interface ListBaseCommands {
      */
     CompletableFuture<String> lpop(String key);
 
+    /**
+     * Returns the index of matching elements inside a list. By default, it will scan the list from
+     * head to tail, looking for the first match of <code>element</code>. If the <code>element</code>
+     * is found, its index is returned. If no match is found, <code>null</code> is returned.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/lpos/">redis.io</a> for details.
+     * @param key The name of the list.
+     * @param element The value to search for within the list.
+     * @return The index of <code>element</code>.
+     * @example
+     *     <pre>{@code
+     * assertEquals(6L, client.rpush("my_list", new String[] {"a", "b", "c", "d", "e", "e"}).get());
+     * assertEquals(4L, client.lpos("my_list", "e").get());
```

Comment:
`assertEquals` and other assert functions are for tests. Use `assert` statement as we do in other examples:
```suggestion
     * assert 6L == client.rpush("my_list", new String[] {"a", "b", "c", "d", "e", "e"}).get();
     * assert 4L == client.lpos("my_list", "e").get();
```
or
```suggestion
     * Long listLen = client.rpush("my_list", new String[] {"a", "b", "c", "d", "e", "e"}).get();
     * assert listLen == 6L;
     * Long position = client.lpos("my_list", "e").get();
     * assert position == 4L;
```
but usually we ommit the first part, so it could be
```suggestion
     * Long position = client.lpos("my_list", "e").get();
     * assert position == 4L;
```